features/rest_owner_role.feature

```diff
@@ -0,0 +1,16 @@
+Feature: As a restaurant owner
+  In order to administer my restaurant (menu, dishes etc.)
+  I need a login that gives me restaurant owner rights
+
+  Background:
+    Given the following restaurant owner credentials exist
+      | email               | password       | password_confirmation | restaurant_owner        |
+      | max@mollerstrom     | 12345678       | 12345678              | true                    |
+
+  Scenario: Restaurant Owner Login
+    Given I visit the landing page
+    And  I click on link "Log In"
+    And I fill in field "E-mail" with "max@mollerstrom"
+    And I fill in field "Password" with "12345678"
+    And I click on button "Log in"
+    Then I should see link "View My Restaurant Dashboard"
```

Comment:
I would like to see a test that makes sure that we are not getting a false positive. Meaning that we should have a user that is NOT an admin and consequently does NOT see the dashboard link.